public/index.php

```diff
@@ -13,7 +13,7 @@
 }
 
 if ($trustedProxies = $_SERVER['TRUSTED_PROXIES'] ?? $_ENV['TRUSTED_PROXIES'] ?? false) {
```

Comment:
If `TRUSTED_PROXIES` contains **`127.0.0.1,$VPC_CIDR,REMOTE_ADDR`** (where `$VPC_CIDR` is replaced with the real CIDR of this VPC) then it looks OK to me

Reply:
Adding the VPC_CIDR won't make any difference since there's REMOTE_ADDR which I guess is looser as a check.
Following the official doc https://symfony.com/doc/current/deployment/proxies.html#but-what-if-the-ip-of-my-reverse-proxy-changes-constantly, I should used a sg to block incoming external traffic (which I do since it's on ECS on a private IP) and use `127.0.0.1,REMOTE_ADDR` to whitelist everything (since it should be legit incoming traffic).

Am I missing something?